pkg/curatedpackages/packagetypes.go

```diff
@@ -0,0 +1,19 @@
+package curatedpackages
+
+import (
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+)
+
+// DisplayablePackage wraps Package to omit undesired members (like Status).
+//
+// This is necessary in part because of https://github.com/golang/go/issues/11939
+// but also because we just don't want to generate a Status section when we're
+// emitting templates for a user to modify.
+type DisplayablePackage struct {
+	*api.Package
+	Status *interface{} `json:"status,omitempty"`
+}
+
+func NewDisplayablePackage(p api.Package) DisplayablePackage {
+	return DisplayablePackage{Package: &p}
```

Comment:
why receive the package by value and then store a pointer?
Whatever is your decision, I would probably do the same thing for both.

Also, it's a bit misleading to return a struct by value that contains a pointer. Usually returning by value tells the caller that that object is _theirs_ and can be copied without side effects. But having a pointer inside breaks that assumption.